indico/modules/events/abstracts/__init__.py

```diff
@@ -84,3 +93,35 @@ class AbstractReviewerRole(ManagementRole):
     name = 'abstract_reviewer'
     friendly_name = _('Reviewer')
     description = _('Grants abstract reviewing rights on an event.')
+
+
+@signals.get_placeholders.connect_via('abstract-notification-email')
+def _get_notification_placeholders(sender, **kwargs):
+    from indico.modules.events.abstracts.placeholders import (
```

Comment:
I think this is one of the cases where `from indico.modules.events.abstracts import placeholders` would be nicer. could even define `__all__` in there and then simply do e.g. this:

```
for name in placeholders.__all__:
    obj = getattr(placeholders, name)
    if isinstance(name, ...):
        yield obj
```